src/main/java/language/English.java

```diff
@@ -0,0 +1,61 @@
+package language;
+
+public class English implements Language {
+
+    public String chooseLanguageMessage() { return "Please choose a language:"; }
+
+    public String repeatLanguageMessage() {
+        return "Invalid language, please repeat:";
+    }
+
+    public String welcomeMessage() {
+        return "*** Welcome to Rock-Paper-Scissors ***";
+    }
+
+    public String nameMessage() {
+        return "Please type your name:";
+    }
+
+    public String moveMessage() {
+        return "Please type your move:";
```

Comment:
It might be handy to have an indication of what the valid moves are here, eg: `Please type your move (Rock/Paper/Scissors)`

Reply:
True! Will do that